Booting/linux-bootstrap-3.md

```diff
@@ -1,20 +1,23 @@
 Kernel booting process. Part 3.
-================================================================================
+==========================================================
```

Comment:
Thank you for fixes, please update it to 80 chars as it was before and I will merge it.